pkg/machines/xmlCreator.js

```diff
@@ -104,7 +104,7 @@ export function getVolumeXML(volumeName, size, format, target) {
     volElem.appendChild(nameElem);
 
     var allocationElem = doc.createElement('capacity');
-    allocationElem.setAttribute('unit', 'MB');
+    allocationElem.setAttribute('unit', 'MiB');
```

Comment:
Libvirt supports both MB and MiB, I was indeed mixing these, which is a real issue. But I think instead of converting everything to {K, M, G}iB it would be better to convert everything to {K, M , G}B since the latter units are more often used. WDYT?

Reply:
Isn't the trend now to move everything {K, M , G}B to {K, M , G}iB? Most "new" stuff uses {K, M , G}iB already and libvirt, while having {K, M , G}B for some old things, also now uses {K, M , G}iB for new stuff.

Reply:
Also in most of cockpit we use MiB/GiB..., in Storage modul especially, so it would be nice to keep it unified.

Reply:
Just checked the storage page. Good then, let's stick to the latter.